doc/content/gateways/mikrotikrouterboard/_index.md

```diff
@@ -20,7 +20,7 @@ There are multiple interfaces to configure the gateway parameters. This page gui
 
 The MikroTik Routerboard wAP LoRa8 kit comes with a PoE adapter. The following image shows the proper connections to the device.
 
-{{< figure src="mikrotik-routerboard-connections.jpg" alt="MikroTik Routerboard wAP LoRa8 kit connections" >}}
+{{< figure src="mikrotik-routerboard-connections.jpg" alt="MikroTik Routerboard wAP LoRa8 kit connections" width=80% >}}
```

Comment:
I'm seeing 45, 50, 55, 65, 70, 80 and 85% in this PR. Is this a percentage of screen width or of the original image? Is there a guideline for the right number?